plugins/tailwind/compiler.ts

```diff
@@ -76,3 +114,27 @@ export async function initTailwind(
 
   return postcss(plugins);
 }
+
+function extractSpecifiers(graph: ModuleGraphJson, projectLocation: string) {
+  return graph.modules
+    .filter((module) =>
+      module.specifier.endsWith(".tsx") &&
+      module.specifier.startsWith(path.dirname(projectLocation))
+    )
+    .map((module) => module.specifier);
+}
+
+function createCustomResolver(
+  imports: ImportMap,
+  baseURL: URL,
+) {
+  return (specifier: string, referrer: string) => {
+    if (
+      specifier.startsWith("./") || specifier.startsWith("../") ||
+      specifier.startsWith("../../")
+    ) {
+      return path.join(path.dirname(referrer), specifier);
+    }
+    return resolveModuleSpecifier(specifier, imports, baseURL);
+  };
+}
```

Comment:
The hint I got from Luca in https://github.com/denoland/deno_graph/issues/364#issuecomment-1909835945 made this significantly better than it was, but I was still having issues with relative imports. So the resolution here is still hacky and obviously won't work if there is more relative nesting in the specifier.

I'll keep looking at this until it's better behaving.